.gitignore

```diff
@@ -1,3 +1,4 @@
 .DS_Store
+.idea
```

Comment:
It's for WebStorm project files, if you don't use WebStorm it makes sense to remove it so I'll take it out.